api_client.py

```diff
@@ -19,27 +19,60 @@ def login(self, email, password):
             Exception: If the server returns a non-200 HTTP status code.
         """
 
-        with open('authenticate_query.graphql', 'r') as file:
+        login_data = self.make_graphql_request("Authenticate", {
+            "email": email,
+            "password": password,
+            "deviceId": "d2900c00-042d-4db2-a329-798fcd2f152e",
```

Comment:
I don't have access to Famly anymore, but are you certain that embedding a hard coded device ID (of your device?) is a good idea, or should one potentially be randomly generated (if it doesn't break anything).

Reply:
This was added in a [previous PR](https://github.com/jacobbunk/famly-fetch/pull/4/files#diff-683655cd50eab99140fe24a5ded1632d937770515acf23f4f45ca67ed95c2640R79), just refactored slightly here.